libraries/AP_RSSI/AP_RSSI.cpp

```diff
@@ -141,7 +141,7 @@ float AP_RSSI::read_receiver_rssi()
         case RssiType::RECEIVER: {
             int16_t rssi = RC_Channels::get_receiver_rssi();
             if (rssi != -1) {
-                return rssi / 255.0;
+                return rssi * 0.00392; // divide by 255
```

Comment:
```suggestion
                return rssi * (1/255f);
```

Same result, but it's much clearer where the constant came from, and will compile to the same thing